src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -88,30 +82,23 @@ def on_time_step(self, event: Event) -> None:
         """
 
         persons = self.population_view.get(event.index)
-        non_household_heads = persons.loc[
-            persons.relation_to_household_head != "Reference person"
-        ]
 
-        # Get subsets of possible simulants that can move
-        persons_who_move_idx = self.randomness.filter_for_rate(
-            non_household_heads.index,
-            self.person_move_rate(non_household_heads.index),
-            "all_movers",
+        # Get subsets of simulants that do each move type on this timestep
+        move_type_probabilities = (
```

Comment:
Don't rename columns here. There's no reason why `self.person_move_rates()` shouldn't return a DataFrame which already has the correct column names. Either fix the backing csv or set the column names properly in `setup` before setting it as the backing data of the `LookupTable`. 

The residual choice column should also be created in setup